website/docs/components/copy/snippet/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,60 @@
+## Usage
+
+### When to use
+
+- To allow users to copy code snippets, such as URLs, tokens, IDs, etc.
```

Comment:
I think someone said something during one of our meetings or in a comment somewhere about possibly not wanting to use the term "allow". (I know I suggested it here.)

So I was thinking that a better, more positive word choice would be "enable".